__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require('supertest');
+const db = require('../db/connection')
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+
+beforeEach(() => {
+    return seed(data);
+});
+afterAll(() => {
+    return db.end();
+});
+
+describe('GET /api/topics', () => {
+    test('should return 200 OK status code and a body of all topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body)
+            .toEqual(expect.arrayContaining(
```

Comment:
this test needs to be refactored - you only need to check if those properties exist, not what their values exactly are - this test will also fail if any of the test data is changed